packages/flutter/lib/src/widgets/default_selection_style.dart

```diff
@@ -41,8 +41,32 @@ class DefaultSelectionStyle extends InheritedTheme {
   const DefaultSelectionStyle.fallback({ super.key })
     : cursorColor = null,
       selectionColor = null,
+      mouseCursor = null,
       super(child: const _NullWidget());
 
+  /// Creates a default selection style that overrides the selection styles in
+  /// scope at this point in the widget tree.
```

Comment:
nit: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph